### PR TITLE
(DOCSP-14159) 2.14.2 release notes

### PR DESCRIPTION
**Pull request by zach-carr**
[Jira](https://jira.mongodb.org/browse/DOCSP-14159)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/zach.carr/DOCSP-14159/release-notes.html#bi-2-14-2)

Hi @rychipman, would you mind reviewing these 2.14.2 release notes?

I pulled what I thought were the relevant notes from [this Jira page](https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=13184&version=30354), but I'm not familiar with BI Connector or how the docs are maintained yet (I have a meeting this afternoon to get up to speed). Please let me know if something looks off, thank you!

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -20,6 +20,28 @@ Release Notes for |bi|
...
+  usage of expansion directives.
+
+- Adds support for ``__rest`` and ``__exec`` in 
+  `YAML <http://yaml.org/>`_ configurations.
+
+Bug Fix
+~~~~~~~
+
+- Fixes an issue in which some queries took significantly longer to run 
+  on |bi| 2.14.
```
Comment (rychipman) [inline source/release-notes.txt:43]:
```suggestion
- Fixes an issue that caused the |bi| to generate inefficient translations for ``in`` and ``or`` expressions.
```

File: source/release-notes.txt [hunk 2]
```diff
@@ -20,6 +20,28 @@ Release Notes for |bi|
...
+
+New Features
+~~~~~~~~~~~~
+
+- Adds the ``--configExpand`` command line option that enables the 
+  usage of expansion directives.
+
+- Adds support for ``__rest`` and ``__exec`` in 
+  `YAML <http://yaml.org/>`_ configurations.
+
```
Comment (rychipman) [inline source/release-notes.txt:38]:
```suggestion
```
Let's leave these out of the changelog until the entire BI-2149 epic is complete

**Review by rychipman — CHANGES_REQUESTED** [review]
two comments. let's also include that detached signatures are now available for BI Connector releases

**Comment by zach-carr** [comment]
@rychipman thanks for reviewing! I made those changes and it's ready for another look whenever you get a moment:

[Staging link (same as above)](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/zach.carr/DOCSP-14159/release-notes.html#bi-2-14-2)

File: source/release-notes.txt [hunk 3]
```diff
@@ -20,6 +20,24 @@ Release Notes for |bi|
...
+-----------
+
+*Released January 27, 2021*
+
+New Features
+~~~~~~~~~~~~
+
+- Detached signatures are now available for |bi| releases.
+
+Bug Fix
```
Comment (rychipman) [inline source/release-notes.txt:35]:
```suggestion
Bug Fixes
```

File: source/release-notes.txt [hunk 4]
```diff
@@ -20,6 +20,24 @@ Release Notes for |bi|
...
+
+|bi| 2.14.2
+-----------
+
+*Released January 27, 2021*
+
+New Features
+~~~~~~~~~~~~
+
+- Detached signatures are now available for |bi| releases.
```
Comment (rychipman) [inline source/release-notes.txt:33]:
```suggestion
- Detached signatures are now available for release archives.
```

File: source/release-notes.txt [hunk 5]
```diff
@@ -20,6 +20,24 @@ Release Notes for |bi|
...
+
+Bug Fix
+~~~~~~~
+
+- Fixes an issue that caused the |bi| to generate inefficient
```
Comment (rychipman) [inline source/release-notes.txt:38]:
```suggestion
- Fixes an issue that reulted in inefficient 
```

**Review by rychipman — APPROVED** [review]
added a few small suggestions to bring the style/phrasing in line with older release notes. LGTM if those are applied

**Pull request merged**